src/styles/components/_product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $color-grey-light;
+  background: $color-white;
+  padding: 32px 16px 16px;
+  font-family: Roboto, Helvetica, sans-serif;
+
+  &__image-container {
+    margin-bottom: 36px;
+    padding: 0 3px;
+  }
```

Comment:
1. Get rid of this container
2. Apply `margin-bottom` to image directly.
3. About `padding: 0 3px`. The whole idea is that the image is centred horizontally, I can easily break your code by adding more width to the card😢 Try making it always centered independent to width.

Reply:
Did all that and centered the image with margin: 0 auto;